.changeset/dull-dots-act.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
+---
+
+adds sideEffects false back to package.json
```

Comment:
I don't think we need a changeset on this PR because it patches unreleased changes

Reply:
removed it, but can't add the label